src/FIO/TransactionBuilder.cpp

```diff
@@ -120,7 +120,12 @@ string TransactionBuilder::createAddPubAddress(const Address& address, const Pri
     const auto apiName = "addaddress";
 
     string actor = Actor::actor(address);
-    AddPubAddressData aaData(fioName, pubAddresses, fee, walletTpId, actor);
+    // convert addresses to add chainCode -- set it equal to tokenCode
```

Comment:
what is tokenCode?

Reply:
What we call coinSymbol

Reply:
Let’s call it token symbol. 

It’s a good idea to avoid any bad naming from other chains

Reply:
done